pyxem/utils/calibration.py

```diff
@@ -271,6 +283,37 @@ def __repr__(self):
     def axes(self):
         return [ax.axis for ax in self.signal.axes_manager.signal_axes][::-1]
 
+    @property
+    def pixel_extent(self):
+        """Return an array with axes [x/y, left/right, pixel_extent], as follows:
+        [
+            # x axis
+            [
+                # left
+                [boundary 1, boundary 2 ...],
+                # right
+                [boundary 1, boundary 2 ...],
+            ],
+            # y axis
+            [
+                # left
+                [boundary 1, boundary 2 ...],
+                # right
+                [boundary 1, boundary 2 ...],
+            ],
+        ]
+        """
+        if self.flat_ewald:
+            extents = []
+            for ax, scale in zip(self.axes, self.scale):
+                left = ax - scale / 2
+                right = ax + scale / 2
+                extent = np.stack((left, right))
+                extents.append(extent)
+            return extents
+        else:
+            return self._pixel_extent
```

Comment:
This should be calculated from the axes_manager similar to above to handle the case of someone slicing the signal.

I'm not sure what the best way to do that is but we could also just assume that it is linear for half a pixel and that should be fine.